locales-pending/dashboard-premium.ftl

```diff
@@ -0,0 +1,160 @@
+premium-badge-label = { -brand-premium }
```

Comment:
Note: we should add a license header to all the FTL files.

```
# This Source Code Form is subject to the terms of the Mozilla Public
# License, v. 2.0. If a copy of the MPL was not distributed with this
# file, You can obtain one at http://mozilla.org/MPL/2.0/.
```

Reply:
Good call: 7e8339939 and dbf502ff9.

Reply:
We should add it also to the ones in `locales-pending` though.

Reply:
Added in 1aaa95cd8 for all existing FTL files that weren't touched in the PR as well. And I figured I'd just commit the removal of trailing spaces that got applied when I saved those files.